ethcore/machine/src/machine.rs

```diff
@@ -96,6 +96,11 @@ impl Machine {
 		self.ethash_extensions.as_ref()
 	}
 
+	/// Get a reference to the transaction filter, if present.
+	pub fn tx_filter(&self) -> Option<&Arc<TransactionFilter>> {
```

Comment:
Why do we need to have it here, since it's not used anywhere, or have I missed smth?

Reply:
Sorry, this is perhaps an unwanted effect of a rebase.